docs/data-sources/database.md

```diff
@@ -0,0 +1,107 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "encore_database Data Source - terraform-provider-encore"
+subcategory: ""
+description: |-
+  Encore provisioned database information
+---
+
+# encore_database (Data Source)
```

Comment:
We call it `sqldb` to disambiguate for when we have other types of databases, so I'd make this `encore_sql_database`

Reply:
+1